examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  title: string;
+};
+
+export const Tabs = ({ children }) => {
```

Comment:
In the example we've been using functions.

```suggestion
export function Tabs({ children }) {
```